test/observer.spec.ts

```diff
@@ -333,12 +334,42 @@ describe('DataLayerObserver unit tests', () => {
 
     observer.handlers[0].fireEvent();
 
-    const [category] = expectParams(globalMock.console, 'log');
+    let [category] = expectParams(globalMock.console, 'log');
     expect((category as PageCategory).primaryCategory).to.eq(
       globalMock.digitalData.page.category.primaryCategory.toUpperCase(),
     );
 
     ExpectObserver.getInstance().cleanup(observer);
+
+    expectNoCalls(globalMock.console, 'log');
+
+    // test multiple operators
+    observer = ExpectObserver.getInstance().create({
+      beforeDestination: [
+        { name: 'toUpper' },
+        { name: 'suffix' }, // NOTE suffix is a built-in operator
```

Comment:
Do you also need to explicitly test this, or is it covered implicitly in `expect(category.primaryCategory_str)`

Reply:
The expect on ‘ category.primaryCategory_str’ is checking that the suffix operator actually suffixes the property (ie a suffixed property exists and it’s set to the corresponding value).  Then there’s this more general ExpectObserver that checks the observer is created correctly.  We def need to check the property in the emitted object from the operator chain, but there’s an opportunity to clean up the test harnesses if it’s not immediately intuitive.

Reply:
I think my comment was about clarity in the tests - maybe just a comment above `expect(category.primaryCategory_str).to.eq(` that states the expectation also covers the suffix operator will indicate that expected suffixing behavior is covered too.